src/rfc9110.ts

```diff
@@ -47,6 +55,24 @@ export function parseWWWAuthenticate(header: string): string[] {
     return matches;
 }
 
+// Consumes data:
+//   PrivateToken challenge="abc...", token-key="123...",
+//   PrivateToken challenge="def...==", token-key="234...",
+//   PrivateToken challenge=ghi..., token-key=345...
+// Parse WWW-Authenticate according to RFC9110 Section 11.6.1 https://www.rfc-editor.org/rfc/rfc9110#section-11.6.1
+export function parseWWWAuthenticate(header: string): string[] {
+    return parseWWWAuthenticateInternal(header, false);
+}
+
+// Consumes data:
+//   PrivateToken challenge="abc...", token-key="123...",
+//   PrivateToken challenge="def...==", token-key="234...",
+//   PrivateToken challenge=ghi...==, token-key=345...
+// Parse WWW-Authenticate according to RFC9110 Section 11.6.1 https://www.rfc-editor.org/rfc/rfc9110#section-11.6.1
```

Comment:
let's include a warning sentence in the doc that spots the deviating difference from the compliant parse rule.

Reply:
updated.